datasets/librispeech_asr/librispeech_asr.py

```diff
@@ -69,6 +75,14 @@ def map_to_array(batch):
         "dev": _DL_URL + "dev-other.tar.gz",
         "train.500": _DL_URL + "train-other-500.tar.gz",
     },
+    "all": {
+        "dev": {retrieve_subset(u): _DL_URL + u for u in ["dev-clean.tar.gz", "dev-other.tar.gz"]},
+        "test": {retrieve_subset(u): _DL_URL + u for u in ["test-clean.tar.gz", "test-other.tar.gz"]},
+        "train": {
```

Comment:
But this is fine. I think it's easy to apply any custom filtering for curriculum learning for early stages in training and then later using the whole dataset.